boards/arm/nucleo_f303re/doc/index.rst

```diff
@@ -0,0 +1,169 @@
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 72 MHz max CPU frequency
+- VDD from 2.0 V to 3.6 V
+- 512 MB Flash
+- 64 + 16 KB SRAM
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (4)
+- Basic Timer
+- Watchdog Timers (2)
+- PWM channels (18)
+- SPI/I2S (2)
+- I2C (3)
+- USART/UART (3/3)
+- USB 2.0 FS with on-chip PHY
+- CAN (2)
+- GPIO with external interrupt capability
+- DMA channels (12)
+- Capacitive sensing channels (18)
+- 12-bit ADC with 40 channels (4)
+- 12-bit D/A converter with two channels
+- Analog comparator (7)
+- Op amp (4)
+- Capacitive sensing 24 channels
+
+
+More information about the STM32F303RE can be found here:
+
+- `STM32F303RE on www.st.com`_
+- `STM32F303RE reference manual`_
+- `STM32F303RE datasheet`_
+
+Supported Features
+==================
+
+The Zephyr nucleo_f302r8 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f303re/nucleo_f303re_defconfig``
+
+Connections and IOs
+===================
+
+The Nucleo F303RE Board has 5 GPIO controllers. These controllers are
+responsible for pin muxing, input/output, pull-up, etc.
+
+Board connectors:
+-----------------
+.. image:: img/nucleo_connectors.png
+   :width: 800px
+   :align: center
+   :height: 619px
+   :alt: Nucleo F303RE connectors
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+The Nucleo F303RE board features an Arduino Uno V3 connector and a ST
+morpho connector. Board is configured as follows:
+
+- UART_2 TX/RX : PA2/PA3 (ST-Link Virtual Port Com)
+- I2C1 SCL/SDA : PB8/PB9 (Arduino I2C)
+- SPI1 CS/SCK/MISO/MOSI : PB6/PB13/PB14/P15 (Arduino SPI)
+- PWM_2_CH2 : PA0
```

Comment:
i2c, spi, pwm not enabled in board.dts and in pinmux file, please remove this line.